### PR TITLE
Update year range to 2016

### PR DESCRIPTION
**Pull request by pra85**

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -91,7 +91,7 @@ module.exports = function (grunt) {
             'FileDescription': WINDOWS_APPNAME,
             'InternalName': BASENAME + '.exe',
             'OriginalFilename': BASENAME + '.exe',
-            'LegalCopyright': 'Copyright 2015 Docker Inc. All rights reserved.'
+            'LegalCopyright': 'Copyright 2016 Docker Inc. All rights reserved.'
```
Comment (thaJeztah) [inline Gruntfile.js]: Should probably be 2015-2016 (or simply "Copyright Docker Inc....")

Comment (thaJeztah) [inline Gruntfile.js]: http://www.contentious.com/2007/01/07/copyright-notice-is-the-year-really-necessary/

**Comment by pra85** [comment]
@thaJeztah  I have removed years from the copyright notice. But I would also suggest getting advise from somebody more knowledgeable about this subject (The reason why I am suggesting this is because, the site you mentioned above - http://www.contentious.com/2007/01/07/copyright-notice-is-the-year-really-necessary/ - does have a year in their footer region, the thing which they claimed to have removed site-wide )

**Comment by thaJeztah** [comment]
Alright if we really want to be sure, then I suggest making it 2015-2016 in Gruntfile.js, and 2014-2015 in the License.

Reason I commented on the Gruntfile.js, was that you changed the year (and that year was the date the file was originally created)

**Comment by pra85** [comment]
I have updated the pull request with the changes you suggested

**Comment by FrenchBen** [comment]
IANAL but LGTM. Ping @jeffdm

**Comment by jeffdm** [comment]
Thanks! Awesome!

One last thing, can you sign your commit?

```
$ git commit --amend -s --no-edit
$ git push -f
```

**Comment by pra85** [comment]
@jeffdm I have signed my commit and updated the pull request

**Comment by jeffdm** [comment]
LGTM, thanks!

**Pull request merged**